app/assets/javascripts/movies.js

```diff
@@ -0,0 +1,19 @@
+$(function() {
+  $('html').on('change', 'select#movie_event_ids.chosen', function(){
```

Comment:
Try to scroll over your select box and look if your JavaScript keep calm or become mad